modules/aws_roles/README.md

```diff
@@ -0,0 +1,177 @@
+# operator role module
+
+Create rosa operator roles and identity provider in an declarative way
+Terraform AWS ROSA STS Roles
+
+In order to deploy [ROSA](https://docs.openshift.com/rosa/welcome/index.html) with [STS](https://docs.openshift.com/rosa/rosa_planning/rosa-sts-aws-prereqs.html), AWS Account needs to have the following roles placed:
+
+* Account Roles (One per AWS account)
+* OCM Roles (For OCM UI, One per OCM Org)
+* User Role (For OCM UI, One per OCM user account)
+* Operator Roles (One Per Cluster)
+* OIDC Identity Provider (One Per Cluster)
+
+This terraform module tries to replicate rosa CLI roles creation so that:
+
+* Users have a declartive way to create AWS roles.
+* Users can implement security/infrastructure as code practices.
+* Batch creation of operator roles.
+
+## Prerequisites
```

Comment:
Does an `AWS CLI` a prerequisite as well?
To run `aws configure` with the account credentials.

Reply:
Added it